collector/stat_linux.go

```diff
@@ -98,11 +98,11 @@ func (c *statCollector) Update(ch chan<- prometheus.Metric) (err error) {
 			// Only some of these may be present, depending on kernel version.
 			cpuFields := []string{"user", "nice", "system", "idle", "iowait", "irq", "softirq", "steal", "guest"}
 			// OpenVZ guests lack the "guest" CPU field, which needs to be ignored.
-			expectedFieldNum := len(cpuFields) + 1
+			expectedFieldNum := len(cpuFields)+1
```

Comment:
I use vim-go bundle, it may have changed this.
I'll run gofmt again and check changes.